### ISSUE TITLE
Updating my app.js & app.css on theme watch/deploy supposedly fails, but reflects success on the actual storefront

### ISSUE THREAD

**Issue by smjeffries-snap** [Shopify/themekit#857]
**Describe the bug**
I am using Laravel Mix to compile all my JavaScript/CSS into one file each.

I run theme watch to detect the newest changes in my compiled JavaScript/CSS (deploy to manually push changes). However, since updating to 1.1.3 this morning, the log keeps reporting failure of the app.js & app.css specifically, even though I am successfully seeing the changes in the browser.

Liquid files still show successful update as expected.

```
11:47:01 [development] processing assets/app.js
[development] (assets/app.js) request failed after 5 retries
```

**To Reproduce**
1. run command 'theme deploy' or 'theme watch' with a new version of a js and/or css file in the assets directory

**Expected behavior**
ThemeKit to report a successful update upon theme watch/deploy.

eg.

```
11:47:01 [development] processing assets/app.js
[development] (assets/app.js) Updated assets/app.js
```

**Environment:**
 - **OS:** Ubuntu 20.04.1 LTS
- **Themekit version:** 1.1.3
 - **Editor:** PhpStorm
 - **Ignore:** What's this for?

**Comment by efekurnaz** [comment]
Same issue here. Development build script was more than 4mb and request keep failing. On production build, ~400kb, it was okay.

Seems like uploading operation is timing out on large files.

To downgrade: `theme update --version=v1.1.2`

**Comment by kellyvaughn** [comment]
Same issue, only fails even on attempting to run `theme watch` where I'm not trying to deploy large files.

**Comment by tanema** [comment]
Hi everyone, I am looking into this. @kellyvaughn is the theme that you are running `theme watch` on large in general?

In the most recent release, I added a bunch of timeout settings to try to make hanging requests fail fast however it might have caused requests for large assets to fail now. I will have a test release for you to try in a moment.

**Comment by davidmartel73** [comment]
Same issue here, @tanema It's only 40kb - sections/header.liquid

![image](https://user-images.githubusercontent.com/61061560/101050474-6f8c7b00-3552-11eb-99fc-52b8e0efe2a0.png)

**Comment by tanema** [comment]
Hello everyone, please try updating to my fix for this issue `theme update --version=v1.1.4-issue857` and let me know if it works.

**Comment by kellyvaughn** [comment]
@tanema This version works for me!

**Comment by davidmartel73** [comment]
@tanema It is working now! Thanks for your quick fix.

**Comment by tanema** [comment]
Thanks for confirming, I will push out 1.1.4 soon as it has a couple quick fixes from 1.1.3 😬

**Comment by smjeffries-snap** [comment]
> Thanks for confirming, I will push out 1.1.4 soon as it has a couple quick fixes from 1.1.3

Mate, you're a legend 😎
